### PR TITLE
Add design elements to portfolio

### PR DESCRIPTION
**Pull request by kanupriyaa**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,74 +2,133 @@
...
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Kanupriya's Portfolio</title>
     <link rel = "stylesheet" href = "style.css">
     <script src = "script.js"></script>
   </head>
   <body>
-
-        <div class = "container navbar">
+
```
Comment (streetdogg) [inline portfolio/src/main/webapp/index.html:10]: trailing white spaces? If so I think it'll be nice to remove them :)

Comment (streetdogg) [reply portfolio/src/main/webapp/index.html:10]: This may actually be a tab.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,74 +2,133 @@
...
         <h1>Work</h1>
+        <p>Previously interned at:</p>
         <div class = "work-info">
             <ol>
-                <li>MathWorks</li>
-                <li>CRIS</li>
-                <li>CodeChef</li>
+                <li>1. MathWorks: Created a search index generator for a set of Simulink Data Model instances using metamodel reflection</li>
+                <li>2. CRIS: Co-developed the CM2S2 App for CMM department using Android Studio</li>
+                <li>3. CodeChef: Created, formulated and modified lesson plans on logical reasoning, problem solving and python programming and executed them.</li>
```
Comment (streetdogg) [inline portfolio/src/main/webapp/index.html]: suggestions: If possibly you can add hyperlinks to the work here. This need not be done now.

**Review by streetdogg — APPROVED** [review]
Looks good to me. Minor comments.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (kanupriyaa) [reply portfolio/src/main/webapp/index.html:10]: Fixed this

**Review by streetdogg — COMMENTED** [review]
Nit: Be mindful of the permissions on the images used. Otherwise, Looks good to me. 

edit: never mind, this seems to be merged already!

**Pull request merged**